core/health/src/test/java/io/thorntail/health/EndpointTest.java

```diff
@@ -48,11 +50,18 @@ public void test() {
         assertThat(checks.size()).isGreaterThan(0);
 
         checks.forEach(check -> {
+            assertThat(((JsonObject) check).getString("name")).isEqualTo("test-check");
             assertThat(((JsonObject) check).getString("state")).isEqualTo("UP");
         });
     }
 
     @Inject
     @Management
     URL url;
+    
+    @Health
+    @Produces
+    public HealthCheck check() {
```

Comment:
I'd say it's more idiomatic to annotate the `TestHealthCheck` class with `@Health` instead of creating a producer. But both should work.